### ISSUE TITLE
Add accessibility properties to table columns

### ISSUE THREAD

**Issue by aleksfront** [ixrock/mobx-react-table-grid#7]
We need to allow columns to be focusable with tab key. And accessibility props should be added also as well.

https://github.com/ixrock/mobx-react-table-grid/assets/9607060/b54fed73-d107-4d0f-ab24-0c7faae804d1

This might work but need to distinct header column from "regular" ones.

<img width="695" alt="Screenshot 2023-05-19 at 16 23 23" src="https://github.com/ixrock/mobx-react-table-grid/assets/9607060/b7ab1e89-8adc-47e3-aace-2e47ff89e632">
